packages/storybook/src/nimble/select/select.stories.ts

```diff
@@ -91,6 +119,10 @@ To display placeholder text within the \`Select\` you must provide an option tha
 Any \`Select\` without a default selected option should provide placeholder text. Placeholder text should always follow the pattern "Select [thing(s)]", for example "Select country". Use sentence casing and don't include punctuation at the end of the prompt.
 `;
 
+const groupedDescription = `
```

Comment:
When elements are used with specific child elements I am moving our docs to a pattern of creating separate stories for those child elements. You can see the pattern in the menu and tab stories now. It allows us to explicitly document the API of the child elements rather than trying to cram a bunch of info in a catch-all control that doesn't match to an actual attribute / slot.

I've [locally started refactoring the select story](https://github.com/ni/nimble/compare/api-docs-inputs?expand=1) to get ready to use some of these patterns but haven't quite written a story for the list option yet (or the group, obviously).

This is mostly a heads up that I will want to change this and that we should coordinate. You are welcome to start following that pattern now, wait for me to come through and adopt it, or plan to adopt it up yourself later once I've laid the groundwork.

Reply:
I'll make a pass at adopting it.

Reply:
This will actually happen in a follow up PR.